### ISSUE TITLE
Vertex order of dodecahendron used for the case 2.

### ISSUE THREAD

**Issue by krips89** [kanezaki/pytorch-rotationnet#4]
Hello,
Is it possible for you guys to provided the order of the vertices of dodecahendron used for generating the views in case 2 (and hence its permutation in vcand file). I tried rendering in the order of the vertices provided here: https://github.com/kanezaki/SHREC2017_track3/blob/master/mvcnn_utils/render_views.m, but it seems that the generated images of ModelNet dataset doesn't look similar to that of the one in the datset, and as a consequence the training does not converge when the vcand.py was used for calculating the next candidate views for for case 2.
Thanks!

**Comment by kanezaki** [comment]
Hello,
The order of the vertices of dodecahendron (written in line 51-75) in render_views.m should work fine with vcand_case2.npy. Please make sure that "ims = render_views('airplane_0001.off')" would generate 20 images in the order like this:
![airplane_0001](https://user-images.githubusercontent.com/740761/52896282-d9665400-3208-11e9-971b-9a6ac2065e4a.png)
If training does not converge, please try a smaller learning rate. Thanks!

**Comment by krips89** [comment]
Can you also provide the details, how you generated the vcand_case2 permutations? 
Thanks!

**Comment by kanezaki** [comment]
OK, here it is.
https://github.com/kanezaki/pytorch-rotationnet/blob/master/generate_case2_rotation_candidates.m
Thanks!
